third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -1719,25 +1719,30 @@ impl Script {
     }
 }
 
+#[derive(Clone)]
+struct StructLayoutInfoCacheItem {
+    struct_layout: MoveStructLayout,
+    node_count: u64,
+    has_identifier_mappings: bool,
```

Comment:
What are node_count and has_identifier_mappings?

Reply:
number of recursive nodes, and whether struct has a field with (aggregator) identifier